fetools-app/src/pages/ShadowGenerator.jsx

```diff
@@ -76,15 +84,15 @@ const ShadowGenerator = () => {
           setNumOfShadows={setNumOfShadows}
           ActiveShadow={ActiveShadow}
           setActiveShadow={setActiveShadow}
+          setRemoveShadow={setRemoveShadow}
         />
       </section>
 
       <section className="lg:mx-48">
         <CodeBlock title="Code" code={"box-shadow"} unit={currentStyle} />
-
-
-       
       </section>
+
+      <section className="lg:mx-48">{/* <GoDeeper/> */}</section>
     </>
   );
 };
```

Comment:
```suggestion
};

```

Simple bump to prompt deploy preview